spec/onlyoffice_file_helper/file_helper_copy_file_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe OnlyofficeFileHelper::FileHelper, '.copy_file' do
+  let(:file_path) { '/tmp/copy_file.ext' }
```

Comment:
I think unique file names are better in this test, maybe something with ruby internal `Tempfile` and `Dir.mktmpdir`